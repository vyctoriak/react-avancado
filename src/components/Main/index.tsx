import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Title>React Avan√ßado</S.Title>
  </S.Wrapper>
)

export default Main
