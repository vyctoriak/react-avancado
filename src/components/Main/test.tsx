import { render, screen } from '@testing-library/react'

import Main from '.'

describe('testing Main component', () => {
  it('should render the heading', () => {
    render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avançado/i })
    ).toBeInTheDocument() //o i serve para NÃO fazer diferença entre maiúsculas ou minúsculas
  })

  /*it('should render correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toMatchSnapshot()
  })*/
})
