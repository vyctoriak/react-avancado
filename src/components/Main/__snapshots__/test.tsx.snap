// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Main component should render correctly 1`] = `
<main>
  <h1>
    React Avan√ßado
  </h1>
</main>
`
